### DISCUSSION TITLE
Deno panicked while trying to install the DeptNews Quarto template

### DISCUSSION THREAD

**Discussion by ptram** [denoland/deno#20722] [Q&A]
Hi,

I was trying to install the mentioned Quarto template in my Mac Monterey, but it panicked.

Quarto is version 1.4.355, Pandoc version 3.1.7.

Unfortunately, I'm not able to "run it with the RUST_BACKTRACE" etc.

--------------------

Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: macos x86_64
Version: 1.33.4
Args: ["/Applications/quarto/bin/tools/x86_64/deno", "run", "--unstable", "--no-config", "--allow-read", "--allow-write", "--allow-run", "--allow-env", "--allow-net", "--allow-ffi", "--no-check", "--importmap=/Applications/quarto/bin/vendor/import_map.json", "/Applications/quarto/bin/quarto.js", "use", "template", "quarto-ext/typst-templates/dept-news"]

thread 'main' panicked at 'could not load platform certs: Custom { kind: Other, error: Error { code: -25262, message: "The Trust Settings Record was corrupted." } }', cli/args/mod.rs:493:41
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by lucacasonato** [comment]
It seems you are using `DENO_TLS_CA_STORE=system`, is that correct?

If so, does following these steps help you https://github.com/cloudflare/wrangler-legacy/issues/1656? Your certificate keychain on macOS seems to be corrupted.
